src/services/codefixes/importFixes.ts

```diff
@@ -1014,6 +1025,10 @@ function getUmdSymbol(token: Node, checker: TypeChecker): Symbol | undefined {
  * @internal
  */
 export function getImportKind(importingFile: SourceFile, exportKind: ExportKind, compilerOptions: CompilerOptions, forceImportKeyword?: boolean): ImportKind {
+    if (compilerOptions.verbatimModuleSyntax && (getEmitModuleKind(compilerOptions) === ModuleKind.CommonJS || importingFile.impliedNodeFormat === ModuleKind.CommonJS)) {
+        // TODO: if the exporting file is ESM under nodenext, or `forceImport` is given in a JS file, this is impossible

```

Comment:
Dynamic import should still work, no? CJS in node next still has access to esm dynamic import, it's just cumbersome to use because promises. Still, we could always wrap everything up into an async iife or something.

Reply:
I think it’s unclear whether these are desirable in auto-imports. First of all, the syntactic form it should take is hard to know. If the usage is already in an async function, no problem. But if not, our best guess is likely going to be very intrusive. And moreover, we should never show a dynamic import of an ESM thing when there’s a CJS equivalent available, and determining what’s equivalent can be fraught. This question isn’t new to this PR; it’s new since nodenext, and we haven’t heard demand for this AFAIK.

Reply:
I think it’s unclear whether these are desirable in auto-imports. First of all, the syntactic form it should take is hard to know. If the usage is already in an async function, no problem. But if not, our best guess is likely going to be very intrusive. And moreover, we should never show a dynamic import of an ESM thing when there’s a CJS equivalent available, and determining what’s equivalent can be fraught. This question isn’t new to this PR; it’s new since nodenext, and we haven’t heard demand for this AFAIK.